test/event_handler_test.exs

```diff
@@ -1,7 +1,13 @@
-defmodule KaufmannEx.Stages.EventHandlerTest do
+defmodule KaufmannEx.Consumer.Stage.EventHandlerTest do
   use ExUnit.Case
-  alias KaufmannEx.TestSupport.MockBus
   alias KafkaEx.Protocol.Fetch.Message
+  alias KaufmannEx.Consumer.GenConsumer
+  alias KaufmannEx.Consumer.Stage.{Producer, Decoder, EventHandler}
```

Comment:
The alias `KaufmannEx.Consumer.Stage` is not alphabetically ordered among its group.